invenio_rdm_records/services/components/verified.py

```diff
@@ -27,4 +43,30 @@ def publish(self, identity, draft=None, record=None):
 
             if not is_verified:
                 # Spawn a task to request moderation.
-                request_moderation.delay(record.parent.access.owner.owner_id)
+                uow.register(
+                    TaskOp(request_moderation, record.parent.access.owner.owner_id)
+                )
+
+
+class ContentModerationComponent(ServiceComponent):
+    """Service component for content moderation."""
+
+    @staticmethod
+    def handler_for(action):
+        """Get the handlers for an action."""
+
+        def _method(self, *args, **kwargs):
+            handlers = current_app.config.get(
+                "RRM_CONTENT_MODERATION_HANDLERS", {}
+            ).get(action, [])
+            for handler in handlers:
+                handler(*args, **kwargs)
+
+        return _method
+
+    update_draft = handler_for("update_draft")
```

Comment:
Tests are failing, I think you need:
```suggestion
    update_draft = ContentModerationComponent.handler_for("update_draft")
```